warehouse-management-service/db/migrations/000003_create_shelf_block_table.up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE IF NOT EXISTS shelf_block(
+    id TEXT PRIMARY KEY DEFAULT gen_random_uuid (),
+    aisle TEXT,
+    rack TEXT,
+    storage_type TEXT
```

Comment:
Might want to set up an [enum](https://www.postgresql.org/docs/current/datatype-enum.html) for `storage_type` to restrict values to a known set.

Reply:
It is not decided what would be the allowed set of values for `storage_type` in the scope of the problem yet. Hence I did not make this an enum.